workitem/link/type_repository.go

```diff
@@ -69,14 +68,21 @@ func (r *GormWorkItemLinkTypeRepository) Create(ctx context.Context, name string
 		return nil, errors.NewInternalError(fmt.Sprintf("Failed to find work item link category: %s", db.Error.Error()))
 	}
 	// Check space exists
-	space := space.Space{}
-	db = r.db.Where("id=?", linkType.SpaceID).Find(&space)
-	if db.RecordNotFound() {
-		return nil, errors.NewBadParameterError("work item link space", linkType.SpaceID)
-	}
-	if db.Error != nil {
-		return nil, errors.NewInternalError(fmt.Sprintf("Failed to find work item link space: %s", db.Error.Error()))
-	}
+
+	//TODO(kwk): Currently I have to disable this because when applying a space
+	//template to a temporary (soft-deleted) space, the space ID cannot be found.
```

Comment:
what about passing a `space` instead of a `spaceID`, then ? you wouldn't have to verify that the space exists, but you'd be sure it does.